packages/fe/pages/about.vue

```diff
@@ -162,6 +162,9 @@ export default {
       }
     }
   }
+  .markdown h5 {
+    margin-bottom: 1.125rem;
+  }
```

Comment:
This should be applied inside the `.markdown` class in `main.scss`. Also all headings should have some inherent spacing, not just `h5`'s